curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-sports-team-cards/63d12fe2a595263e8f5084f7.md

```diff
@@ -0,0 +1,465 @@
+---
+id: 63d12fe2a595263e8f5084f7
+title: Step 15
+challengeType: 0
+dashedName: step-15
+---
+
+# --description--
+
+To check if the `Object.freeze()` method is working, you can try to assign a new value to one of the existing properties.
+
+Below the `Object.freeze(myFavoriteFootballTeam)`, add the following lines of code:
+
+```js
+
+myFavoriteFootballTeam.team = "USA"
+console.log(myFavoriteFootballTeam.team)
+
+```
```

Comment:
I think added this example more so for the `console.log` part because I wasn't sure how much exposure to `console.log` campers have had to this point. 

But we could rewrite it to remove the example and just have instructions here instead